src/css/table.css

```diff
@@ -66,6 +66,15 @@ table.tableblock > tbody th {
   padding-bottom: var(--base-space);
 }
 
+/* Activate the grid=none option for Asciidoc */
+table.tableblock.grid-none > tbody td,
+table.tableblock.grid-none > tbody th {
+  border-bottom: 0;
+  padding-left: var(--base-small-space);
+  /* padding-top: var(--base-space); */
```

Comment:
How come we've commented this out?

Reply:
Ah, yes; 

I copied it from the block above and I forgot to take it out.

I'll fix it and then push the change.